source/_topics/splitting_configuration.markdown

```diff
@@ -196,6 +196,22 @@ We offer four advanced options to include whole directories at once.
 - `!include_dir_merge_list` will return the content of a directory as a list by merging all files (which should contain a list) into 1 big list.
 - `!include_dir_merge_named` will return the content of a directory as a dictionary by loading each file and merging it into 1 big dictionary.
 
+These work recursively, for example;
+
+```bash
+.
+└── config
```

Comment:
Please change `config` to `.homeassistant` since it is the default name for the config directory.